chart/templates/xnat-web-statefulset.yaml

```diff
@@ -53,11 +59,33 @@ spec:
             - /bin/sh
             - -c
             - |
-              /usr/local/bin/wait-for-postgres.sh
               /usr/local/tomcat/bin/catalina.sh run
       initContainers:
+        - name: wait-for-postgres
+          image: {{ include "xnat.image" . }}
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+            - name: PGPASSWORD
+              {{- if .Values.postgresql.auth.existingSecret }}
```

Comment:
`postgresql.auth.existingSecret` [defaults to an empty string](https://github.com/bitnami/charts/blob/6fef27c5c4c0da7ec01455bb332ecfc92535b5fc/bitnami/postgresql/values.yaml#L164)